user.php

```diff
@@ -0,0 +1,61 @@
+<?php
+	include("connection.php");
+	/**
+	* @author: mfsi_krishnadev
+	* @class: validating user
+	*/
+	class user extends db_connection
+	{
+		
+		/*
+		 * @access: public
+		 * @param : email-id, password 
+		 * @return type: none
+		*/
+
+		function check($mail_id, $pass)
+		{
+
+			$email_id = $this->test_input($mail_id);
+			$password = md5($pass);
+
+
+			$select = "SELECT id, CONCAT(first_name, ' ',last_name)as name FROM employee
+				WHERE email_id = '$email_id' AND password = '$password'";
+
+			$query = mysqli_query($this->connect, $select);
+			$row = mysqli_num_rows($query);
+
+			if($row == 1)
+			{
+				session_start();
+				$row = mysqli_fetch_assoc($query); 		
+				$user_id = $row['id'];
+				$user_name = $row['name'];
```

Comment:
No need to store $row values in a variable and then store in session.
directly can do
$_SESSION['user_id'] = $row['id'];
$_SESSION['user_name'] =  $row['name'];

but before that check if $row = mysqli_fetch_assoc($query); is returning any valid result or not before assigning the result value to session.